install_config/persistent_storage/persistent_storage_csi.adoc

```diff
@@ -328,6 +328,8 @@ spec:
 <1> Replace with `cloud.conf` for your OpenStack deployment, as described in
  xref:../../install_config/configuring_openstack.adoc#configuring-openstack-variables[OpenStack configuration]. For example, the Secret can be generated using the `oc create secret generic cloudconfig --from-file cloud.conf --dry-run -o yaml`.
 
+<2> Optionally, add `nodeSelector` to the CSI driver pod template to configure the nodes on which the CSI driver should be started. Only nodes matching the selector will be able to run pods that use volumes served by the CSI driver. Without `nodeSelector`, the driver will run on all nodes in the cluster.
```

Comment:
@jhoyt-rh - Hi Joan - this is missing: https://github.com/openshift/openshift-docs/pull/11606/files#diff-fa532ad4b01ebfd61474800bc6c0c948L256.

The <2> is like a footnote that is referencing content that has changed earlier in the book. See Jan's original PR.

Reply:
Please edit the text to remove "should" and the future tense.
s/volumes served/volumes that are served